packages/yew-macro/tests/hook_macro/use_prepared_state-fail.stderr

```diff
@@ -1,14 +1,16 @@
-error: expected closure
- --> tests/hook_macro/use_prepared_state-fail.rs:6:38
+error: this hook takes 2 arguments but 1 argument was supplied
+ --> tests/hook_macro/use_prepared_state-fail.rs:6:5
   |
 6 |     use_prepared_state_with_closure!(123)?;
-  |                                      ^^^
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  |
+  = note: this error originates in the macro `use_prepared_state_with_closure` (in Nightly builds, run with -Z macro-backtrace for more info)
 
-error: You must specify a return type for this closure. This is used when the closure is omitted from the client side rendering bundle.
- --> tests/hook_macro/use_prepared_state-fail.rs:8:38
+error: expected closure
+ --> tests/hook_macro/use_prepared_state-fail.rs:8:55
   |
 8 |     use_prepared_state_with_closure!(|_| { todo!() }, 123)?;
-  |                                      ^^^^^^^^^^^^^^^
+  |                                                       ^^^
 
```

Comment:
This test also needs to be updated as it is asserting the return type is required.